packages/mgt-components/src/components/mgt-contact/mgt-contact.ts

```diff
@@ -199,22 +199,22 @@ export class MgtPersonCardContact extends BasePersonCardSection {
     }
 
     contentTemplate = html`
-      ${compactParts.map(p => this.renderContactPart(p))}
-    `;
+       ${compactParts.map(p => this.renderContactPart(p))}
```

Comment:
What's going on with your `html` blocks? they all seem to be getting one extra space of indenting throwing them off the two space indent step that we have defined.

Yes, it's picky, but there's a lot change here that doesn't need to be.